config/iocs/Makefile

```diff
@@ -0,0 +1,23 @@
+EPICS_REPO ?= file:///afs/slac/g/pcds/vol2/svn/pcds
+
+all:
+	@echo "Checking out and updating IOCs..."
+	@for fn in ../notepad/*.cfg; do \
+		hutch=$$(echo $$fn | sed s/.*ioc-// | sed s/-pcdsdevices.cfg// ); \
+		echo "Hutch: $$hutch"; \
+		ioc_path=$$EPICS_REPO/epics/trunk/ioc/$$hutch/pvNotepad/current; \
+		if (svn ls $$ioc_path > /dev/null 2>&1); then \
+			echo "* Found IOC in SVN"; \
+			svn checkout $$ioc_path $$hutch || svn resolve --accept theirs-full $$hutch; \
```

Comment:
This is a godsend for people like me who can't remember more than one VCS at a time

Reply:
I'm also guilty of this - working back from "_Git for SVN users cheatsheet_" is my new go-to...